src/make/cards.js

```diff
@@ -45,6 +45,11 @@ function before() {
       || /draft/.test(card.text))
       card.rarity = 'special'
 
+  for (card of raw.FRF.cards)
+    // Includes all possible lands except Crucible of Spirit Dragon, the only rare land.
+    if ((/Land/.test(card.type)) && (!(/Rare/.test(card.rarity))))
+      card.rarity = 'special'
+
```

Comment:
turn the comment into code, and use less parens:

    if (/Land/.test(card.type) && (card.name !== 'Crucible of the Spirit Dragon'))